src/core/db/models.py

```diff
@@ -110,3 +111,33 @@ class Category(ContentBase):
 
     def __repr__(self):
         return f"<Category {self.name}>"
+
+
+class AdminUser(Base):
+    __tablename__ = "admin_users"
+
+    email: Mapped[str] = mapped_column(String(48), unique=True)
+    first_name: Mapped[str] = mapped_column(String(64), nullable=True)
+    last_name: Mapped[str] = mapped_column(String(64), nullable=True)
+    password: Mapped[str] = mapped_column(String(128))
+    last_logon: Mapped[date]
```

Comment:
Давай не будет ошибки переносить) 
```suggestion
    last_login: Mapped[date]
```